tests/generic/test_wrapper.py

```diff
@@ -0,0 +1,80 @@
+"""
+Test for the wrapper
+Also serve as a generic test
+"""
+
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import mean_squared_error as mse
+import tensorflow as tf
+
+from xplique.attributions import KernelShap, Lime, Occlusion
+from ..utils import generate_dataset, generate_linear_model
+
+
+def test_wrapper():
+    """
+    Test the wrapping of a sklearn model, by comparing it to a keras model.
+    Both are linear regression model with really close weights.
+    We expect the explanations on those models to be close as well.
+    """
+    dataset, target = generate_dataset(10000)
+
+    x_train, x_test, y_train, _ = \
+        train_test_split(dataset, target, test_size=0.1, shuffle=True)
+
+    libraries = [
+        "sklearn",
+        "keras",
+    ]
+
+    models = {
+        lib: generate_linear_model(x_train, y_train, lib)
+        for lib in libraries
+    }
+
+    # we want both models to be close enough
+    y_preds = [model(x_test) for model in models.values()]
+    assert mse(y_preds[0], y_preds[1]) <= 0.01
+
+    # Define explainers
+    model_explainers = {}
+    for library, model in models.items():
+        model_explainers[library] = {
+            "KernelShap": KernelShap(
+                model,
+                nb_samples=200,  # 2000
+                ref_value=0.0
+            ),
+            "Lime": Lime(
+                model,
+                nb_samples=200,  # 2000
+                ref_value=0.0,
+                distance_mode="cosine",
+                kernel_width=1.0
+            ),
+            "Occlusion": Occlusion(
+                model,
+                patch_size=1,
+                patch_stride=1,
+                occlusion_value=0.0
+            ),
+        }
+
+    # cast a subset to tf format
+    subset_len = 100
+    inputs_tf = tf.cast(x_test[:subset_len], tf.float32)
+    targets_tf = tf.ones((subset_len, 1))
+
+    explanations = {}
+
+    # compute explanations for all methods
+    for library, explainers in model_explainers.items():
+        explanations[library] = {}
+        for method, explainer in explainers.items():
+            # compute explanation for a given method
+            explanations[library][method] = explainer(inputs_tf, targets_tf)
+
+    for method in explanations[libraries[0]].keys():
+        expl0 = explanations[libraries[0]][method]
+        expl1 = explanations[libraries[1]][method]
+        assert mse(expl0, expl1) <= 0.01
```

Comment:
you can use `almost equal` if you want here (but not an obligation).

Reply:
Now `almost_equal()` is used